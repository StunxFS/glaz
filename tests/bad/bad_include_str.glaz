fn main() {
	_ = include_str!("nonexistent.file");
}
