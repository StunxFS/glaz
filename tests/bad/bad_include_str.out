tests/bad/bad_include_str.glaz:2:20: error: cannot include `nonexistent.file` (does not exist)
     | 
   2 | 	_ = include_str!("nonexistent.file");
     | 	                 ^^^^^^^^^^^^^^^^^^
glaz error: could not compile `bad_include_str`, aborting due to previous error.
